manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.1.0
```

```suggestion
PackageVersion: 4.1
```
This must match Control Panel.

You'll also need to change `manifests/t/Telegram/TelegramDesktop/4.1.0` to `manifests/t/Telegram/TelegramDesktop/4.1`